testgrid/cmd/updater/main.go

```diff
@@ -300,8 +365,36 @@ func AppendColumn(headers []string, grid *state.Grid, rows map[string]*state.Row
 
 const elapsedKey = "seconds-elapsed"
 
-func ReadBuild(build Build) (*BuildResult, error) {
-	br := BuildResult{
+var re = regexp.MustCompile(`.+/junit(_[^_]+)?(_\d+-\d+)?(_\d+)?\.xml$`)
```

Comment:
don't need dropPrefix with a better regex:

    regexp.MustCompile(`.+/junit(?:_([^_]+))?(?:_(\d+-\d+))?(?:_(\d+))?\.xml$`)

Reply:
Better to keep the regex easier to read